android/.settings/org.eclipse.buildship.core.prefs

```diff
@@ -1,11 +1,11 @@
 arguments=
 auto.sync=false
 build.scans.enabled=false
-connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)
+connection.gradle.distribution=GRADLE_DISTRIBUTION(VERSION(7.4.2))
```

Comment:
curious why we are updating the versions here and below?

Reply:
@Kiruel

Reply:
Probably my vscode doing weird things, should I remove this ?

Reply:
If it's not needed then sure! 😃

Reply:
@liyiy Pushed without it.